### PR TITLE
better match the mocks for op shelves

### PR DESCRIPTION
**Pull request by spasovski**
Mock:

SSs (image border exists to show its dimensions only):

### CODE REVIEW

File: src/media/css/feed/shelves.styl [hunk 1]
```diff
@@ -121,3 +162,26 @@ shelf_curve_color($color) {
...
+    }
+    [data-shelf-landing] .feed-landing-header-desktop {
+        display: block;
+    }
+}
+
+@media $base-desktop {
+    .feed-shelf-landing-inner > div {
+        left: 50%;
+        margin-left: -($desktop-content / 2 - 130);
```
Comment (ngokevin) [inline src/media/css/feed/shelves.styl]: too magic.

File: src/media/css/feed/shelves.styl [hunk 2]
```diff
@@ -63,6 +63,47 @@ shelf_curve_color($color) {
...
+        top: 40px;
+        width: 100%;
+    }
+    h1,
+    .feed-landing-description {
+        text-align: left;
+    }
+}
+.feed-shelf-info {
+    float: left;
```
Comment (ngokevin) [inline src/media/css/feed/shelves.styl]: can't flexbox it?

**Comment by ngokevin** [comment]
Looks like for me, which is too far left, and the text is a bit too close to the top of the curve.

screen shot 2015-03-06 at 1 24 30 pm

**Comment by ddurst** [comment]
Centering looks a little funny. What happens if there's no description text, like in that 800x1280 setting?

**Comment by ngokevin** [comment]
screen shot 2015-03-06 at 1 26 43 pm

**Comment by ngokevin** [comment]
I was able to do this in a jiffy by adding ```position: relative; bottom: 30px``` to the img and inner content of the curve. And then ```padding-top: 30px``` to the top of the desktop curve header. And then reduce the ```padding-bottom``` of the desktop curve header to 10px.

screen shot 2015-03-06 at 1 28 56 pm

**Comment by spasovski** [comment]
Good finds folks. I'll see if a softer approach is feasible.

**Comment by ngokevin** [comment]
Going to close this as I think the solution is simpler. Reopen if we find this is all necessary.

File: src/media/css/feed/shelves.styl (hunk 2 above)
Comment (chuckharmston) [inline src/media/css/feed/shelves.styl]: Yeah, flexbox is made for things like this. I've been using it quite a bit lately, holler if you'd like a sounding board for brainstorming.

**Pull request closed without merging**